Corona-Warn-App/src/main/java/de/rki/coronawarnapp/datadonation/analytics/server/DataDonationAnalyticsServer.kt

```diff
@@ -19,14 +19,15 @@ class DataDonationAnalyticsServer @Inject constructor(
             Timber.d("Response code: %d", it)
         }
 
-        return when (code) {
-            204 -> Timber.d("Analytics upload completed successfully")
-            400, 401, 403 -> {
+        return when {
+            response.isSuccessful -> Timber.d("Analytics upload completed successfully")
+            code in listOf(400, 401, 403) -> {
```

Comment:
this is different issue, response code is 200 now, I use isSuccessful instead to cover all possible success codes